### ISSUE TITLE
tab_model function not printing interaction effects

### ISSUE THREAD

**Issue by htrier** [strengejacke/sjPlot#646]
Hello, I am using tab_model to print the output of some brm models and I can't seem to get the function to include interaction effects in the html file. When I print the model summary using summary() in the console I see all interaction and main effects listed under population-level effects, but in the tab_model result only main effects are listed. Am I doing something wrong in the call to tab_model or could there be a better way to tell the function to print all effects? My code calling tab_model:

sjPlot::tab_model(model1,show.ci50=FALSE,show.ci=0.95,dv.labels=c('Full model output'),file=file.path(paste0(save_path,filename,'95CI.html')),title=html_title)

**Comment by strengejacke** [comment]
Do you have a reproducible example?

**Comment by htrier** [comment]
Yes sorry for not including the reproducible example in the first instance. Here's a toy example; I've attached an xlsx file with the first 25 rows of my data. When I input the data into this model:

`testmodel <- brm(y ~ (var1+mo(var2)+var3)*(var4+var5)+((var1+mo(var2)+var3)*(var4+var5)|ParticipantID),data=df,iter=3000,control=list(adapt_delta=0.9))`

and then view the output using `summary(testmodel)`, I see all main and interaction effects reported. However, when I print the html file using tab_model:

`sjPlot::tab_model(testmodel,show.ci50=FALSE,show.ci=0.95,dv.labels=c('Full model output'),file=file.path(paste0('/mypath/testhtml_95CI.html')),title="Relationships among task variables and behavior")
`
I see only the main effects reported. I really appreciate your advice thanks!

[test_data.xlsx](https://github.com/strengejacke/sjPlot/files/4719292/test_data.xlsx)

**Comment by strengejacke** [comment]
Any reason for closing this? Could you solve this issue?

**Comment by htrier** [comment]
HI

**Comment by htrier** [comment]
Hi, I had closed the issue because I thought I'd solved it but it turns out I haven't. I ran the same tab_model example on a different computer and was able to print the interactions to the html file, but I still have the same issue on my main computer and I can't figure out why tab_model operates differently on the two computers. Perhaps I need to update sjPlot? I'm checking whether the two computers are using the same package versions, and oddly the computer on which tab_model works is using earlier versions of some packages (e.g. sjmis and sjlabelled).
